source/directives/rhTable.js

```diff
@@ -150,13 +153,13 @@ module.exports = [ "$compile", "$rootScope", "ngDialog", function ( $compile, $r
 
                 scope.canCreate = function ()
                 {
-                    if( scope.definition.meta.canCreate === false )
+                    if( utils.runIfFunc( scope.definition.meta.canCreate ) === false )
                     {
                         return false;
                     }
                     else if( scope.definition.meta.canCreate && scope.definition.meta.canCreate.max )
                     {
-                        return scope.list.filter( i => !i.meta || !i.meta.deleted ).length
+                        return scope.list.filter( function ( i ) { return !i.meta || !i.meta.deleted; } ).length
```

Comment:
The babelify transform that runs as part of browserify ignores code in node_modules and arrow functions break uglify when it runs for production distribution, and they should be transformed already anyway. I started setting up this project to create an ES5 file and decided to just leave it for later since it was easier to just not use an arrow function here. So, something to figure out later.